src/modules/organizations/organizations.service.ts

```diff
@@ -64,41 +64,41 @@ export class OrganizationsService {
       const limit = query.limit ? parseInt(query.limit, 10) : 10;
 
       const qb = this.entityManager
-        .createQueryBuilder(Organization, 'organization')
+        .createQueryBuilder(Organization, "organization")
         .select([
-          'organization.id',
-          'organization.name',
-          'organization.description',
-          'organization.website',
-          'organization.created_by',
-          'organization.updated_by',
-          'organization.created_at',
-          'organization.updated_at',
+          "organization.id",
+          "organization.name",
+          "organization.description",
+          "organization.website",
```

Comment:
@vaibhavsTekdi 
Single quotes preferred unless there's a strong reason to use double quotes. check your editor config with Tushar.